src/react/hooks/__tests__/useBackgroundQuery.test.tsx

```diff
@@ -8411,16 +8453,51 @@ describe.skip("type tests", () => {
 
       subscribeToMore({
         document: subscription,
-        updateQuery: (queryData, { subscriptionData }) => {
+        updateQuery: (
+          queryData,
+          {
+            subscriptionData,
+            subscriptionVariables,
+            variables,
+            complete,
+            previousQueryResult,
+          }
+        ): UnmaskedVariablesCaseData => {
```

Comment:
I'm curious, is this needed? This might indicate a code smell if the type of `updateQuery` doesn't use the unmasked type. If this is working without the type, I'd remove this and let the type of `updateQuery` dictate this:

```suggestion
        ) => {
```